### PR TITLE
feat: strong linking description

### PR DESCRIPTION
**Pull request by danil-pavlov**
* [KT-67589](https://youtrack.jetbrains.com/issue/KT-67589): This PR adds a section on Obj-C symbols always requiring strong linking

### CODE REVIEW

File: docs/topics/native/native-objc-interop.md [hunk 1]
```diff
@@ -208,6 +210,17 @@ Objective-C does not support packages in a framework. If the Kotlin compiler fin
 which have the same name but different packages, it renames them. This algorithm is not stable yet and can change between
 Kotlin releases. To work around this, you can rename the conflicting Kotlin classes in the framework.
 
+#### Strong linking
+
+Whenever you use an Objective-C class in the Kotlin source, it's marked as a strongly linked symbol. The resulting build
+artifact mentions related symbols as strong external references.
+
+This means that the app tries to link symbols during the launch dynamically, and if they are unavailable, the app crashes.
+Symbols might not be available on a particular device or OS version. It happens even if symbols were never used.
```
Comment (SvyatoslavScherbina) [inline docs/topics/native/native-objc-interop.md:219]: I would rewrite it as follows:
The crash happens even if symbols were never used. Symbols might be unavailable on a particular device or OS version.

This seems a bit less ambiguous to me.

File: docs/topics/native/native-objc-interop.md [hunk 2]
```diff
@@ -208,6 +210,17 @@ Objective-C does not support packages in a framework. If the Kotlin compiler fin
...
+
+To work around this issue and avoid "Class not found" errors, use a Swift or Objective-C wrapper that checks
```
Comment (SvyatoslavScherbina) [inline docs/topics/native/native-objc-interop.md:221]: > "Class not found"

Did you mean "Symbol not found"?

File: docs/topics/native/native-objc-interop.md (hunk 1 above)
Comment (danil-pavlov) [reply docs/topics/native/native-objc-interop.md]: thank you! that works

**Review by SvyatoslavScherbina — APPROVED** [review]

**Review by koshachy — APPROVED** [review]
Nice job!
LGTM!

**Pull request merged**